app/Actions/User/Notify.php

```diff
@@ -28,8 +27,7 @@ public function do(Photo $photo): void
 
 		$users = $users
 			->unique('id', true)
-			->whereNotNull('email')
-			->where('id', '!=', AccessControl::id());
+			->whereNotNull('email');
```

Comment:
I assume the old condition was there to prevent users from receiving emails for photos which they have added themselves. This seems to be a change in behavior. Why?

Reply:
I didn't think much.